### PR TITLE
feat/storybook-7-upgrade-15

### PR DESCRIPTION
**Pull request by panvourtsis**

### CODE REVIEW

File: src/components/ThemeProvider/ThemeProvider.tsx [hunk 1]
```diff
@@ -19,7 +20,10 @@ export type ThemeProviderProps = {
 const deepMergeTheme = (newTheme: DeepPartial<Theme>, theming: 'dark' | 'semantic'): Theme =>
   merge(theme(theming), pick(newTheme, keys(theme(theming))));
 
-const ThemeProvider: React.FCC<ThemeProviderProps> = ({ theme = {}, children }) => {
+const ThemeProvider: React.FC<React.PropsWithChildren<ThemeProviderProps>> = ({
```
Comment (kostasdano) [inline src/components/ThemeProvider/ThemeProvider.tsx:23]: this could probably be FCC

File: src/components/Toast/Toast.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ export const isNotificationTypes = (type: string): type is NotificationTypes =>
   return ['success', 'error', 'warning', 'info'].includes(type);
 };
 
-const Toast: React.FCC<ToastProps> = ({
+const Toast: React.FC<React.PropsWithChildren<ToastProps>> = ({
```
Comment (kostasdano) [inline src/components/Toast/Toast.tsx:42]: this could probably be FCC

**Review by kostasdano — APPROVED** [review]

File: src/components/Toast/Toast.tsx (hunk 2 above)
Comment (panvourtsis) [reply src/components/Toast/Toast.tsx]: fixed here https://github.com/Orfium/orfium-ictinus/pull/718/commits/b8d5f77f8605a2c62ea17e696ae816d9a509dc14

File: src/components/ThemeProvider/ThemeProvider.tsx (hunk 1 above)
Comment (panvourtsis) [reply src/components/ThemeProvider/ThemeProvider.tsx]: fixed here https://github.com/Orfium/orfium-ictinus/pull/718/commits/b8d5f77f8605a2c62ea17e696ae816d9a509dc14

**Pull request merged**